earth_enterprise/src/fusion/portableglobe/servers/fileunpacker/build_and_test.py

```diff
@@ -25,18 +25,145 @@
 
 import distutils.sysconfig
 import os
+import re
 import shutil
+import subprocess
 import sys
 import util
 
 
+def get_cc_version(cc_path):
```

Comment:
I'm sure you've already thought of this, but is this code duplicated from somewhere? Is there any way to eliminate the duplication?

Reply:
Yes, I was looking at that.  There is some utility code, but it's specific to things like WSGI scripts, or Portable Globe CGI scripts.

I could create a Python module near the top-level, and add it to `PYTHONPATH` for scons, and Pontable Server build scripts.

Maybe we can get some votes, or other proposals? (@tst-lsavoie, @tst-dsheremata, @tst-basilhuffman, @tst-ccamp)